gulpfile.js

```diff
@@ -22,27 +26,53 @@ var paths = {
  *
  ***************************************/
 
+// full list of rules at: https://github.com/sasstools/sass-lint/tree/develop/docs/rules
+gulp.task('lint', function() {
+    return gulp.src(paths.src.sass + '**/*.s+(a|c)ss')
+        .pipe(sasslint({
+            options: {
+                formatter: 'stylish',
+                'merge-default-rules': true,
+            },
+            files: {ignore: paths.src.sass + 'vendor/*.scss'},
+            rules: {
+                'no-ids': 2,
+                'no-color-literals': 2,
+                'bem-depth': 2,
+                'property-sort-order': 0,
```

Comment:
We previously reviewed WordPress and Druapl declaration order, I believe we decided on the following:

- Display
- Positioning
- Box model
- Colors and Typography
- Other

We don't have to follow this, but we should decide on one and declare it with PropertySortOrder?

Reply:
I had forgotten what the decision was so I really just put some sample rules in there to test they were working. 

I've since added a .sass-lint.yml file to hold all of the rules, so let's abandon this pull request and I'll resubmit another with the understanding that the linting rules are to be reviewed. Then maybe we can discuss the linting rules so we have some agreement and revise them at that point.

Reply:
This rule will be needed, but probably just need to decide on an declaration order that we want to follow instead of default alphabetical order?

Reply:
Agreed, we will need it. And I agree, I feel like alphabetical is less useful than the groupings you mentioned above. I'll see if I can get that into the lint rules file.